Exercise5.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class Exercise5{
+
+        public static void main(String args[]){
+                Scanner scanner = new Scanner(System.in);
+                String string = scanner.nextLine();
+                wordList(string);
+        }
+
+        public static void wordList(String str){
```

Comment:
Ngoại trừ hàm main ra thì tất cả các hàm, mình đều phải đặt tên bắt đầu bằng 1 động từ. Ở đây, mình cũng nên đặt word ở dạng plural!

```suggestion
        public static void listWords(String str){
```